README.md

```diff
@@ -1,18 +1,77 @@
-# rugs
+# Atomist 'rugs'
 
 [![Build Status](https://travis-ci.org/atomist/rugs.svg?branch=master)](https://travis-ci.org/atomist/rugs)
+[![Slack Status](https://join.atomist.com/badge.svg)](https://join.atomist.com/)
 
-## Testing
+[Node][node] module [`@atomist/rugs`][rugs] that brings in all the
+dependencies you need to write Rugs using [TypeScript][ts].  See
+the [Atomist Documentation][docs] for more information.
 
-npm install -G typescript
-npm test
+[node]: https://nodejs.org/en/
+[rugs]: https://www.npmjs.com/package/@atomist/rugs
+[ts]: https://www.typescriptlang.org/
+[docs]: http://docs.atomist.com/
 
-## Releasing
+This module contains some helper classes and interfaces for writing
+Rugs, but its main purpose is to bring in all the dependencies you
```

Comment:
I wouldn't say that bringing in dependencies is its main purpose...but it _is_ handy.